pkg/grpc/api.go

```diff
@@ -1470,22 +1474,17 @@ func (a *api) SetActorRuntime(actor actors.Actors) {
 }
 
 func (a *api) GetMetadata(ctx context.Context, in *emptypb.Empty) (*runtimev1pb.GetMetadataResponse, error) {
```

Comment:
The returned content is missing `ID`,`ActiveActorsCount`

Reply:
There are many errors in the previous implementation of  `GetMetadata` , which should be modified by referring to HTTP API

Reply:
Already support.